lib/ruby_lsp/requests/document_link.rb

```diff
@@ -63,6 +101,61 @@ def gem_paths
             lookup
           end, T.nilable(T::Hash[String, T::Array[String]]))
         end
+
+        sig { returns(T.nilable(T::Hash[String, T::Hash[Symbol, String]])) }
+        def rails_documents
+          @rails_documents ||= T.let(begin
+            return {} unless RAILTIES_VERSION
+
+            $stderr.puts "Fetching Rails Documents..."
+            response = Net::HTTP.get_response(URI("#{RAILS_DOC_HOST}/v7.1.1/js/search_index.js"))
+
+            # If the version's doc is not found, e.g. Rails main, it'll be redirected
+            # In this case, we just fetch the latest doc
+            if response.code == "302"
```

Comment:
Instead of checking for the response of the http request, is there a way to determine through the gem's version that it's pointing to main?